src/crypto.ts

```diff
@@ -195,4 +200,41 @@ export default class Crypto {
       decodeBase64(formSecretKey)
     )
   }
+
+  /**
+   * Download and decrypt the given attachments.
+   * @param formSecretKey Secret key as a base-64 string
+   * @param decryptParams The params containing encrypted content and information.
+   * @returns The decrypted attachments if successful. Else, null will be returned.
+   * @throws {MissingPublicKeyError} if a public key is not provided when instantiating this class and is needed for verifying signed content.
+   */
+  downloadAndDecryptAttachments = async(formSecretKey: string, decryptParams: DecryptParams): Promise<DecryptedAttachments | null> => {
+    let decryptedRecords: DecryptedAttachments = {}
+
+    const attachmentRecords: EncryptedAttachmentRecords = decryptParams.attachmentDownloadUrls ?? {}
+    const decryptedContent = this.decrypt(formSecretKey, decryptParams)
+    if (decryptedContent === null) return null
+
+    for (let fieldId in attachmentRecords) {
+      const downloadResponse = await axios.get(attachmentRecords[fieldId], { responseType: 'json' })
+      if (downloadResponse.status !== 200) return null
+
+      let data = downloadResponse.data
```

Comment:
could we add a type annotation here or on the `axios.get`? this would allow for readers to know what the expected shape of data is supposed to be - currently, this is typed as `any` but later on, `data.encryptedFile.binary` is accessed, which causes a typeError if `encryptedFile` is `undefined`